tests/PoliceLogin.yaml

```
@@ -42,4 +42,20 @@ appId: com.anonymous.PoliceMobileApp
+
+# DATA OVERVIEW TEST -------------------------------------------------------------------
+# open data overview
+- tapOn: "Profil"
+
+# verify data is visibke 
+- assertVisible: "Ime"
+- assertVisible: "Prezime"
+- assertVisible: "OIB"
+- assertVisible: "E-mail"
+
+# test accuracy 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)